impl/src/test/java/org/jboss/arquillian/warp/impl/utils/TestShrinkWrapUtils.java

```diff
@@ -44,5 +46,7 @@ public void testMultipleUse() throws ClassNotFoundException {
 
         assertNotNull(nestedArchive.get("/org/junit/Test.class"));
         assertNotNull(nestedArchive.get("/org/junit/Ignore.class"));
+
+        classLoader.close();
```

Comment:
Use `try-with-resources` since ShrinkWrapClassLoader implements [AutoCloseable](https://docs.oracle.com/javase/8/docs/api/java/lang/AutoCloseable.html).